### PR TITLE
adds support for local api connections

### PR DESCRIPTION
**Pull request by fredrike**

### CODE REVIEW

File: tellduslive.py [hunk 1]
```diff
@@ -76,23 +76,54 @@ def __init__(self,
...
-            token,
-            token_secret)
+        if host is not None:
+            self._token = token
+            self._session.headers = {"Authorization": "Bearer %s" % token}
+            self._api_url = "http://%s/api/" % host
+        else:
+            self._session.auth = OAuth1(public_key,
+                    private_key, token, token_secret)
+            self._api_url = API_URL
```
Comment (molobrakos) [inline tellduslive.py]: I guess we could rename the API_URL constant later to TELLDUS_LIVE_API_URL or something.

File: tellduslive.py [hunk 2]
```diff
@@ -76,23 +76,54 @@ def __init__(self,
...
         self._state = {}
 
+    def get_authorize_url(host, app):
```
Comment (molobrakos) [inline tellduslive.py]: should this be marked with `@staticmethod`?

File: tellduslive.py [hunk 3]
```diff
@@ -76,23 +76,54 @@ def __init__(self,
...
+        try:
+            r = requests.put("http://%s/api/token" % host, data={'app': app}).json()
+            return r['authUrl'], r['token']
+        except:
+            """Could not get url and token, return"""
+            return "", ""
+
+    def authorize_local_api(host, token):
```
Comment (molobrakos) [inline tellduslive.py]: is this also a staticmethod?

Comment (fredrike) [reply tellduslive.py]: Yes!

File: tellduslive.py (hunk 2 above)
Comment (fredrike) [reply tellduslive.py]: Yes!

**Pull request merged**